### PR TITLE
Solving issue 149:

### PR DESCRIPTION
**Pull request by chrom13**
Fixing the package name.
Issue: https://github.com/Platzi-Master-C9/booking-services/issues/149

### CODE REVIEW

File: packages/notifications-system/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "notifications-system",
```
Comment (JoseNoriegaa) [inline packages/notifications-system/package.json:2]: Package names have to match the pattern `@booking-services/your_package_name`, so your package name should be @booking-`services/notifications-system`

File: packages/notifications-system/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# `message`
```
Comment (JoseNoriegaa) [inline packages/notifications-system/README.md:1]: Can you update this file to match your package name?

**Review by JoseNoriegaa — CHANGES_REQUESTED** [review]
There are still many references to the messages package that should be updated to "notifications system" or something like that.

**Review by Alejandrocuartas — COMMENTED** [review]
Did you run the 'lint' script before committing?

**Comment by JoseNoriegaa** [comment]
@Alejandrocuartas I think we don't need this PR anymore since this was fixed as part of another PR.

**Pull request closed without merging**